documentation/ansible-playbook/modules/ansible-playbook-running-the-ansible-playbook.adoc

```diff
@@ -21,5 +21,6 @@ With the `playbook.yml` set and in-place, run the `playbook.yml`
 
 [source,bash]
 ----
+$ export ANSIBLE_CONFIG=./ansible.cfg
```

Comment:
I guess it would be nice to explain why this is explicitely needed (or at least a link to the docs :D)

Reply:
PR's accepted! ;-)